common/src/main/java/cc/tweaked/prometheus/ServerMetrics.java

```diff
@@ -29,12 +29,26 @@ public static void onServerStart(MinecraftServer server) {
         var ticking = ServerMetrics.toTick = new ArrayList<>();
         var registry = new MetricContext(server, collectorRegistry, ticking::add);
 
-        ComputerCollector.register(registry);
-        ComputerFieldCollector.register(registry);
-        ThreadGroupCollector.register(registry);
+        var computercraftLoaded = false;
+        try {
+            Class.forName("dan200.computercraft.impl.ComputerCraftAPIImpl");
```

Comment:
```suggestion
            Class.forName("dan200.computercraft.api.ComputerCraftAPI");
```

Just because this one is guaranteed to never change.